TKGame/TKGame.cs

```diff
@@ -139,8 +145,11 @@ protected override async void Update(GameTime gameTime)
             //Do if not paused
             if (!paused)
             {
+                EntityManager.Update(gameTime, spriteBatch, currentStage);
+                sword.Update(sword);
```

Comment:
This is what we want to avoid, this is cluttering the TKGame (World) class/functions. The Sword isn't directly owned by the game, wouldn't the Player own the Sword/Weapon component that could be have weaponComponent.Update() called on it in the Player's main update function?

Reply:
Yah, I moved the weapon into the player class and called everything from there.